docs/design/agent-skills/agent-skills.md

```
@@ -0,0 +1,798 @@
+  ┌─────────────────┐
+  │ LLM uses skill  │──► Follows instructions with loaded resources
+  │ instructions    │
+  └─────────────────┘
+```
+
+### Configuration
+
+Skills are configured by specifying paths in `lightspeed-stack.yaml`. Two forms are supported:
+
+**Option A: Directory of skills** (recommended for most deployments)
+```yaml
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)